src/js/components/sortable.js

```
@@ -31,180 +61,183 @@ export default {
-            scroll && setTimeout(() => window.scroll(window.pageXOffset, scroll), 5);
-        }
+            if (insertTarget && placeholder === insertTarget) {
+                return;
+            }
 
-    },
+            if (sortable !== previous) {
+                previous.remove(placeholder);
+                data.moved = target;
+            } else {
+                delete data.moved;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)